1.0/tools/optimize-for-production.md

```diff
@@ -0,0 +1,268 @@
+
+    vulcanize elements.html
+
+The output will be printed to stdout in the terminal.
+
+To write the output to a file, use the `--output` or `-o` flag:
+
+    vulcanize -o elements.vulcanized.html elements.html
+
+No additional configuration is necessary. elements.vulcanized.html now contains a version of elements.html with all imports inlined and dependencies flattened. Paths to any URLs are automatically adjusted for the new output location, with the exception of those set in JavaScript.
+
+You can pass additional options to vulcanize in the form of flags. For a full list of supported flags, see [the official Vulcanize documentation](https://github.com/Polymer/vulcanize#using-vulcanize-programmatically).
+
+Here’s the same example from above, but this time we’ll tell Vulcanize to strip comments and inline any scripts or CSS.
+
+    vulcanize -o elements.vulcanized.html elements.html --strip-comments --inline-scripts --inline-css
+
+### Use vulcanize with gulp
+
+Although Vulcanize does a great job of flattening imports, you may have an existing build system setup that needs to uglify/minify your code or run your CSS through a preprocessor. This section shows you how to add vulcanize to gulp using  [gulp-vulcanize](https://github.com/sindresorhus/gulp-vulcanize).
+
+*Using Grunt?* You can use the [grunt-vulcanize]() task. It supports a similar set of options [maybe link to README?]
+{: .alert .alert-info }
+
+To add vulcanize to our build process:
+
+Install `gulp-vulcanize`
+
+    npm install --save-dev gulp gulp-vulcanize
+
+Require the vulcanize module in your gulpfile and add a task to run it.
+
+    var gulp = require('gulp');
+    var vulcanize = require('gulp-vulcanize');
+
+    gulp.task('vulcanize', function() {
+      return gulp.src(app/elements/elements.html')
+        .pipe(vulcanize())
+        .pipe(gulp.dest('dist/elements'));
+    });
+
+    gulp.task('default', ['vulcanize']);
+
+This sample assumes your project has a single `elements.html` file that imports your other web component dependencies.
+{: .alert .alert-info }
+
+You should now be able to run `gulp` and vulcanize your dependencies.
+
+To configure the task with the same `stripComments`, `inlineScripts`, and `inlineCss` options from above, pass them to the vulcanize task in a configuration object:
+
+    gulp.task('vulcanize', function() {
+      return gulp.src(app/elements/elements.html')
+        .pipe(vulcanize({
+          stripComments: true,
+          inlineScripts: true,
+          inlineCss: true
+        }))
+        .pipe(gulp.dest('dist/elements'));
+    });
+
+Depending on the structure of your app, it may make sense to break it into a few small vulcanized bundles, instead of inlining everything into one file. To prevent specific imports from being inlined in your bundle, use the `excludes` option and pass it an array of file paths or regexes.
```

Comment:
Does this sound better:

Depending on the structure of your app, it may make sense to break it into a few small Vulcanized bundles, instead of inlining everything into one file. This technique can cut down on loading times as it enables you to only load the elements for the current section. To prevent specific imports from being inlined in your bundle, use the `excludes` option and pass it an array of file paths or regexes.